### ISSUE TITLE
Editor not display with toolbarinline option is true

### ISSUE THREAD

**Issue by Sophikitis** [froala/KMSFroalaEditorBundle#107]
Hello,

So before I ask you all my questions, I use EasyAdmin.

My first problem, when I change the "toolbarInline" option to true in my "kms_froala_editor.yaml" file (fig1), the editor does not display (fig2) whereas if I change it to false, it displays but without the toolbar being displayed online. (fig3)
Do you know where the problem may come from? 
fig1 :
![Capture d’écran 2019-11-08 à 18 29 27](https://user-images.githubusercontent.com/33518344/68497926-ea06a080-0255-11ea-8376-c9fc4a16c949.png)
fig2:
<img width="200" alt="Capture d’écran 2019-11-08 à 18 29 46" src="https://user-images.githubusercontent.com/33518344/68497852-ca6f7800-0255-11ea-842b-4ece2dd3dbbf.png">

fig3:
<img width="200" alt="Capture d’écran 2019-11-08 à 18 28 49" src="https://user-images.githubusercontent.com/33518344/68497849-c9d6e180-0255-11ea-9978-6de6496d907e.png">

Then, I wanted to know if it is possible when uploading images to take the id of the article as a parameter for example to define a different upload path for each article.
For example, for the article with id 1, we will have an upload path: /upload/blog/1/

thank you for your help

**Comment by sguionni** [comment]
Hello,
do you have any error in the console ?
I don't use EasyAdmin but something can be in conflict.

**Comment by sguionni** [comment]
For you questions about image upload, you can find your answers in the [official documentation](https://www.froala.com/wysiwyg-editor/docs/concepts/image/upload).

**Comment by Sophikitis** [comment]
Hi, 
Thank you for your answer.
I don't have any errors in the console, but from my research it seems that the "inline" option is only available for those who have a license. But in the end, it doesn't bother me because we can then configure the toolbar, so it's okay! 

I don't go through formbuilder, in the easyadmin configuration file, I give it the type "**froalaEditorType**". (fig1)

![Capture d’écran 2019-11-13 à 13 37 50](https://user-images.githubusercontent.com/33518344/68764426-f6ef0f80-061a-11ea-8c80-f84665fa0ae9.png)

However, I use the "**imageUploadUrl**" options in the file "**kms _froala_editor.yaml**" which allows me to intercept when an image is uploaded through my BlogController. (Fig 2 & 3) .
![Capture d’écran 2019-11-13 à 13 43 09](https://user-images.githubusercontent.com/33518344/68764850-c6f43c00-061b-11ea-9532-d17ae004bcdd.png)
![Capture d’écran 2019-11-13 à 13 44 33](https://user-images.githubusercontent.com/33518344/68764870-d07da400-061b-11ea-82bc-6d1062f8df6b.png)


I also use "**ImageUploadParams**" which allows to add an additional parameter to the request.

And that's where I block it, because I can't dynamically recover the id of my article.

To be as clear as possible, here is the expected behavior for the index ( I have to change, I know  :) ) function of my BlogController:
- I get the id of the article that is passed in parameter
- I concatenate the url with the id to upload all the images in the folder specific to the article
- and I return the link in the answer.

![Capture d’écran 2019-11-13 à 13 49 18](https://user-images.githubusercontent.com/33518344/68765477-2737ad80-061d-11ea-9b7d-e9bd12c3ed89.png)

**Comment by sguionni** [comment]
Hello,
i'm sorry but i don't know how to use EasyAdmin.
Parameters seems to be statics, and you need to change it dynamically (like form builder).

**Comment by Sophikitis** [comment]
It's okay, I managed to do what I wanted.

In the file easy_admin.yaml, I added the addition of a "custom controller" which is specific to the "articles" entity (in my case)


![Capture d’écran 2019-11-28 à 17 25 38](https://user-images.githubusercontent.com/33518344/69822434-a674d600-1206-11ea-8dc7-5ce778785f0b.png)


In my controller I extend my class with EasyAdmin.

I overload the createEntityFormBuilder function which will return the form view in the editing and creation actions.

by getting the builder instance back, I can modify the form. In the example, I modify the "body" field by passing it the "FroalaEditorType" type and some options.
In my case, it is the "imageUploadFolder" and "imageUploadPath" options that interest me. It is from fields that I will be able to define a predefined and unique route for each item I will create

![Capture d’écran 2019-11-28 à 17 33 30](https://user-images.githubusercontent.com/33518344/69822727-606c4200-1207-11ea-867b-5b1c5be87ec2.png)

Finally I return the modified instance of $builder which will be used in the edit and new actions of easyAdmin.

As I defined "custom views" for the edit action and new I just have to extend in twig the template of the targeted action of easyAdmin.

For example, for the edit view: 
![Capture d’écran 2019-11-28 à 17 42 32](https://user-images.githubusercontent.com/33518344/69822920-c789f680-1207-11ea-9c40-072e98cc6ff8.png)

Here is the result: 
![Capture d’écran 2019-11-28 à 17 53 51](https://user-images.githubusercontent.com/33518344/69823039-1768bd80-1208-11ea-8b41-afb7d30f5d15.png)


even if it's not perfect yet, it works perfectly.

Thank you for your help and for taking the time to answer :)
